_books/ion-1-1/src/modules/system_module.md

```diff
@@ -39,18 +51,19 @@ The Ion 1.1 System Symbol table _replaces_ rather than extends the Ion 1.0 Syste
 | 14 | `macro_table`                   |
 | 15 | `symbol_table`                  |
 | 16 | `module`                        |
-| 17 | `retain`                        |
-| 18 | `export`                        |
-| 19 | `catalog_key`                   |
+| 17 | ~~retain~~                      |
+| 18 | export                          |
```

Comment:
Did you mean to remove the backticks for this one?

Reply:
Nope. Thanks for catching that.